Detections/MultipleDataSources/ExtensiveAWSComputeResurseDeployements.yaml

```diff
@@ -0,0 +1,54 @@
+id: 9e457dc4-81f0-4d25-bc37-a5fa4a17946a
+name: Extensive AWS Compute Resurse Deployements
+description: |
+  'This detection focused on Suspicious deployment of AWS EC2 resource (virtual machine) scale sets was detected. This behavior might indicate that the threat actor is deploying computing resources for cryptocurrency mining activities.This detection centers around identifying suspicious instances of AWS EC2 resource deployment, particularly scale sets. Such behavior raises concerns of potential threat actor involvement, potentially indicative of efforts to deploy computing resources for the purpose of cryptocurrency mining activities.
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AWS
+    dataTypes:
+      - AWSCloudTrail
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  -  Impact
+relevantTechniques:
+  - T1496
+query: |
+  // Retrieve AWS CloudTrail events generated within the last day
+  AWSCloudTrail
+  | where TimeGenerated >= ago(1d)

```

Comment:
Again not needed as it matched the query period value

Reply:
removed